src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -385,7 +386,7 @@ const copyingNotebookMessage = div({ style: { paddingTop: '2rem' } }, [
 ])
 
 const NotebookEditorFrame = ({ mode, notebookName, workspace: { workspace: { namespace, name, bucketName } }, cluster: { clusterName, clusterUrl, status, labels } }) => {
-  console.assert(status === 'Running', 'Expected notebook runtime to be running')
+  console.assert(_.includes(status, usableStatuses), 'Expected notebook runtime to be running or updating')
```

Comment:
Nit:
```suggestion
  console.assert(_.includes(status, usableStatuses), `Expected notebook runtime to be one of: ${usableStatuses}`)
```
I think this will do what I expect, but please test to make sure.